services/core-api/app/api/verifiable_credentials/manager.py

```diff
@@ -87,6 +97,87 @@ def offer_newest_amendment_to_current_permittee(permit_amendment_guid: str,
+                        and mpa.mine_party_appt_type_code = 'PMT'
+                        and mpa.deleted_ind = false
+
+                        group by pa.permit_amendment_guid, pa.description, pa.issue_date, pa.permit_amendment_status_code, mpa.deleted_ind, pmt.permit_no, mpa.permit_id, poe.party_guid, p.party_name, poe.name_text, poe.registration_id
+                        order by pmt.permit_no, pa.issue_date;
+
+                       """).fetchall()
+    current_app.logger.warning("Num of results from query to process:" +
+                               str(len(permit_amendment_query_results)))
+
+    traction_service = TractionService()
+    public_did_dict = traction_service.fetch_current_public_did()
+    public_did = "did:indy:bcovrin:test:" + public_did_dict["did"]
+    public_verkey = public_did_dict["verkey"]
+
+    current_app.logger.warning("public did: " + public_did)
+
+    records: List[Tuple[dict, PermitAmendmentOrgBookPublish]] = [] # list of tuples [payload, record]
+
+    for row in permit_amendment_query_results:
+        pa = PermitAmendment.find_by_permit_amendment_guid(row[0], unsafe=True)
+        pa_cred = VerifiableCredentialManager.produce_untp_cc_map_payload(public_did, pa)
+        current_app.logger.warning("bcreg_uri=" +
+                                   str(pa_cred.credentialSubject.issuedTo.identifiers[0].
+                                       verificationEvidence.credentialReference) +
+                                   ", for permit_amendment_guid=" + str(row[0]))
+
+        payload_hash = md5(pa_cred.json(by_alias=True).encode('utf-8')).hexdigest()
+        existing_paob = PermitAmendmentOrgBookPublish.find_by_unsigned_payload_hash(
+            payload_hash, unsafe=True)
+
+        if existing_paob:
+            #this hash has already been seen, do not make new record or publish
+            continue
+
+        paob = PermitAmendmentOrgBookPublish(
+            permit_amendment_guid=row[0],
+            party_guid=row[1],
+            unsigned_payload_hash=payload_hash,
+        )
+        records.append((pa_cred, paob))
+
+    # send to traction to be signed
+    for cred_payload, record in records:
+        signed_cred = traction_service.sign_jsonld_credential_deprecated(
+            public_did, public_verkey, cred_payload)
+        if signed_cred:
+            record.signed_credential = json.dumps(signed_cred)
+            record.sign_date = datetime.now()
+        record.save()
+
+    return [record for payload, record in records]
+
+
+def publish_all_pending_vc_to_orgbook():
+    records_to_publish = PermitAmendmentOrgBookPublish.find_all_unpublished(unsafe=True)
+
+    for record in records_to_publish:
+        current_app.logger.warning("NOT sending cred to orgbook")
+        # resp = requests.post(ORGBOOK_W3C_CRED_POST, record.signed_credential)
+        # assert resp.status_code == 200, f"resp={resp.json()}"
+        current_app.logger.warning(record.signed_credential)
```

Comment:
??
This looks maybe like it was a testing method? The commented out lines seem to be doing the work.

Reply:
There is no API to publish to yet. that functionality needs to be added to Orgbook, so it's a stub for now.